websockets.cabal

```diff
@@ -81,7 +82,9 @@ Library
     random            >= 1.0    && < 1.2,
     SHA               >= 1.5    && < 1.7,
     text              >= 0.10   && < 1.3,
-    entropy           >= 0.2.1  && < 0.4
+    entropy           >= 0.2.1  && < 0.4,
+    streaming-commons >= 0.1.15,
+    safe-exceptions   >= 0.1.4.0
```

Comment:
safe-exceptions is tricky :)
I found that when there is `throw` in pure code then those exceptions leave some `ARR_WORDS`.
There is no easy test for that: start a lot of connections, kill the client, repeat - observe memory growth, and see  `ARR_WORDS` portion growing forever.
Normal termination is fine, all gets GCd.

My bet it is some GHC bug (i.e. exceptions should free forign memory ;) right? )